File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.java
class Solution {
    public int[] leftRightDifference(int[] nums) {
        int n = nums.length;
        int rSum = 0, lSum =0;
        for(int i:nums) rSum+=i;
        // rSum -= nums[0];
        for(int i=0;i<n;i++){
            int temp = nums[i];
            rSum -= temp;
            nums[i] = Math.abs(rSum - lSum);
            lSum+=temp;
        }
        return nums;
    }
}

File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.py
class Solution(object):
    def leftRightDifference(self, nums):
        n = len(nums)
        rightSum = 0
        leftSum = 0
        ans = [0]*n
        for n in nums:
            rightSum+=n
        
        for i,n in enumerate(nums):
            rightSum-=n
            ans[i]=abs(rightSum-leftSum)
            leftSum+=n
        
        return ans

        """
        :type nums: List[int]
        :rtype: List[int]
        """
        